about.html

```diff
@@ -0,0 +1,147 @@
+          The Tech Guru
+          <span class="normal-font">2022</span>.
+        </h1>
+        <p class="main-container__content-info">
+          Scale image and video annotation with pixel-level accuracy by
+          expertly-trained and professionally-managed data annotators.<br />
+          text and audio annotation with a team skilled at understanding and
+          interpreting complex, nuanced language. <br />Optimize your business
+          operations with a flexible, managed workforce for data processing and
+          other back-office tasks.
+        </p>
+        <div>
+          <p class="main-container__content-date">
+            <span class="normal-font">2022.03.15 </span>(MON) ~
+            <span class="normal-font">20 </span>(SAT)
+          </p>
+          <p class="location">@Nakuru, Kisiwa Mall, Kenya</p>
+        </div>
+      </section>
+    </header>
+    <!-- Header Section End -->
+    <!-- About Section Start -->
+    <section class="about">
+      <div class="about-logo">
+        <h2>The Tech Guru Logo</h2>
+        <div class="row"></div>
+        <p>Ther Tech Guru Platform was created for .</p>
+        <img src="./images/TechGuru-Logo.png" alt="logo" />
+      </div>
+      <div class="line"></div>
+      <h2 class="past-conferences__heading">See the past B2B Conferences</h2>
+      <div class="row"></div>
+      <p class="past-conferences__desc">
+        Take a look at the last two B2B Conference which took place in AICC in
+        in Nairobi and also Waterback Hotel in Thika Kenya .
+      </p>
+      <div class="conference">
+        <div class="conference-container twenty">
+          <div class="transparent">
+            <div class="visible">
+              <h1 class="conference-container__head">2021</h1>
+              <p class="conference-container__desc">
+                The Tech Guru 2021 in AICC Nairobi
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="conference-container nineteen">
+          <div class="transparent">
+            <div class="visible">
+              <h1 class="conference-container__head">2020</h1>
+              <p class="conference-container__desc">
+                Tech Guru In Thika Waterback Hotel
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- About Section End -->
+
+    <!-- Partner Section Start -->
+    <section class="about-partner" id="partner">
+      <div class="container _container">
+        <h2 class="titleNumbre">Partners</h2>
+        <div class="numbersContent">
+          <span class="partner">Facebook</span>
+          <span class="partner">Google</span>
+          <span class="partner">Coinbase</span>
+          <span class="partner">Crypto</span>
+          <span class="partner">Pinterest</span>
+        </div>
+      </div>
+    </section>
+    <!-- Partner Section End -->
+
+    <!-- Footer Section Start -->
+    <footer class="footer">
```

Comment:
- [ ] According to the design the footer needs to be same on the Home and About page which you have made correctly but you have also cloned the Partner section on both pages. As per the design given by Microverse there is no partner section on the About page.

**Cloned**: 
![image](https://user-images.githubusercontent.com/58504655/181952642-e17fdfd3-2e81-43df-9933-927128e6ed24.png)

**Original**:
![image](https://user-images.githubusercontent.com/58504655/181952814-443099f1-647a-4850-a7b8-1e5495886273.png)